htdocs/includes/tecnickcom/tcpdf/tcpdf.php

```diff
@@ -19243,7 +19243,7 @@ protected function openHTMLTagHandler($dom, $key, $cell)
 							// convert URL to server path
 							$imgsrc = str_replace(K_PATH_MAIN, K_PATH_URL, $imgsrc);
 						}
-					} elseif (($imgsrc[0] === '/') AND !empty($_SERVER['DOCUMENT_ROOT']) AND ($_SERVER['DOCUMENT_ROOT'] != '/')) {
+					} elseif (!file_exists($imgsrc) && ($imgsrc[0] === '/') AND !empty($_SERVER['DOCUMENT_ROOT']) AND ($_SERVER['DOCUMENT_ROOT'] != '/')) {
```

Comment:
$imgsrc seems to be an array so file_exists on an array parameter when expected a string is strange.